Makefile

```
@@ -170,7 +170,7 @@ build/nginx: build/commons images/nginx/Dockerfile
 build/nginx-drupal: build/nginx images/nginx-drupal/Dockerfile
 build/varnish: build/commons images/varnish/Dockerfile
 build/varnish-drupal: build/varnish images/varnish-drupal/Dockerfile
-build/varnish-persistent: build/commons images/varnish/Dockerfile
+build/varnish-persistent: build/varnish images/varnish/Dockerfile
 build/varnish-persistent-drupal: build/varnish images/varnish-drupal/Dockerfile
```

`build/varnish-persistent-drupal` depends on `lagoon/varnish-drupal`
https://github.com/amazeeio/lagoon/blob/master/images/varnish-persistent-drupal/Dockerfile#L2

```suggestion
build/varnish-persistent-drupal: build/varnish-drupal images/varnish-drupal/Dockerfile
```